test/acceptance/features/createServiceClaim.feature

```diff
@@ -0,0 +1,72 @@
+Feature: Create a service claim with label selector
+
+    Scenario: Create a service claim
+        Given Primaza Cluster "primaza-main" is running
+        When On Primaza Cluster "primaza-main", Resource is created
+        """
+        apiVersion: primaza.io/v1alpha1
+        kind: RegisteredService
+        metadata:
+          name: primaza-rsdb
+          namespace: primaza-system
+        spec:
+          constraints:
+            environments:
+            - stage
+          serviceClassIdentity:
+            - name: type
+              value: psqlserver
+            - name: provider
+              value: aws
+          serviceEndpointDefinition:
+            - name: host
+              value: mydavphost.io
+            - name: port
+              value: "5432"
+            - name: user
+              value: davp
+            - name: password
+              value: quedicelagente
+            - name: database
+              value: davpdata
+          sla: L3
+          """
+        And On Primaza Cluster "primaza-main", Primaza ClusterContext secret "primaza-km" is published
+        And On Primaza Cluster "primaza-main", Resource is created
+        """
+        apiVersion: primaza.io/v1alpha1
+        kind: ClusterEnvironment
+        metadata:
+            name: primaza-main
+            namespace: primaza-system
+        spec:
+            environmentName: stage
+            clusterContextSecret: primaza-km
+        """
+        And On Primaza Cluster "primaza-main", RegisteredService "primaza-rsdb" state will eventually move to "Available"
+        And On Primaza Cluster "primaza-main", Resource is created
+        """
+        apiVersion: primaza.io/v1alpha1
+        kind: ServiceClaim
+        metadata:
+          name: sc-test
+          namespace: primaza-system
+        spec:
+          name: some-name
+          serviceClassIdentity:
+          - name: type
+            value: psqlserver
+          - name: provider
+            value: aws
+          serviceEndpointDefinitionKeys:
+          - username
```

Comment:
Fail the claim if the key is missing.

Reply:
I'd propose to write failure reasons into Status.Condition to help debugging failing claims

Reply:
@filariow Do you mean to use [metav1.Condition](https://kubernetes.io/docs/reference/generated/kubernetes-api/v1.26/#condition-v1-meta) object? If so, can we do it in a separate PR? It requires 5 mandatory fields, for example, it could be something like this:
```
  - type: Ready
    status: 'False'
    reason: 'MissingKey'
    message: 'Key specified in the service end point definition keys is not available in the registered service'
    lastTransitionTime: '2021-01-20T17:00:00Z'
```

Reply:
Created a story: https://issues.redhat.com/browse/APPSVC-1296